src/elements/call-out/src/stories.tsx

```diff
@@ -1,7 +1,10 @@
 /** @jsx jsx */
-import * as React from 'react'
+import { Fragment } from 'react'
 import { jsx } from '@emotion/core'
-import { boolean, select, text } from '@storybook/addon-knobs'
+import { boolean, color, select, text } from '@storybook/addon-knobs'
+import { PaletteProvider } from '@uswitch/trustyle-utils.palette'
+
+import theme from '../../../utils/theme-selector'
```

Comment:
I _think_ this won't work correctly when changing theme - better to get the theme using the `useThemeUI()` hook

Reply:
This works because it's inside the `stories`. I can only use the `useThemeUI()` hook in the component itself